python/tests/kernel/test_ir_operations.py

```diff
@@ -13,22 +13,25 @@
 import cudaq
 from cudaq import spin
 
-
 skipIROperationsForEagerMode = pytest.mark.skipif(
     os.getenv("CUDAQ_PYTEST_EAGER_MODE") == 'ON',
     reason="test_ir_operations only tests MLIR mode of execution")
 
+
 @pytest.fixture(autouse=True)
 def do_something():
     cudaq.enable_jit()
     yield
-    if cudaq.is_jit_enabled(): cudaq.__clearKernelRegistries()
+    if cudaq.is_jit_enabled():
+        cudaq.__clearKernelRegistries()
     cudaq.disable_jit()
 
+
```

Comment:
Edification question: I notice there are a lot of blank lines added in this PR. Is this something in a python coding convention someplace?

Reply:
Yep, I applied code formatting with `yapf` to all changed files, including tests (usually tests are skipped, but I prefer to have it uniform.)